flask_datadog.py

```diff
@@ -169,12 +172,21 @@ def after_request(self, response):
         if self.config['DATADOG_RESPONSE_AUTO_TAG']:
             self.add_request_tags(['status_code:%s' % (response.status_code, )])
 
-        # Emit our timing metric
-        self.statsd.timing(self.config['DATADOG_RESPONSE_METRIC_NAME'],
+        tags = self.get_request_tags()
+
+        # Emit timing metric
+        self.statsd.timing(self.config['DATADOG_RESPONSE_TIME_METRIC_NAME'],
                            elapsed,
-                           self.get_request_tags(),
+                           tags,
                            self.config['DATADOG_RESPONSE_SAMPLE_RATE'])
 
+        # Emit response size metric
+        size = len(response.data)
+        self.statsd.increment(self.config['DATADOG_RESPONSE_SIZE_METRIC_NAME'],
```

Comment:
This might be better as `self.statsd.histogram()`, doing increment would mean that we would be tracking the total response bytes per time period, e.g. "bytes per second" as opposed to "request response size". Although, I can see the value in someone wanting both metrics.

Also, with `histogram`, we'll get the nice `.count`, `.max`, `.avg`, `.95percentile`, `.median` computed metrics as well.